File: Store.BusinessLogic/Models/PrintingEditions/PrintingEditionModelItem.cs
using System.Collections.Generic;
using Store.BusinessLogic.Models.Base;
using Store.BusinessLogic.Models.Authors;
using Store.DataAccess.Entities.Enums;

namespace Store.BusinessLogic.Models.PrintingEditions
{
    public class PrintingEditionModelItem : BaseModel
    {
        public long Id { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public decimal Price { get; set; } = 0;
        public string Image { get; set; }
        public Enums.PrintingEditions.Currency? Currency { get; set; }
        public Enums.PrintingEditions.PrintingEditionType? Type { get; set; }
        public IList<AuthorModelItem> Authors { get; set; } = new List<AuthorModelItem>();
    }
}

File: Store.View/src/app/shared/models/printing-edition/printing-edition.model.item.ts
import { BaseItemModel } from 'src/app/shared/models/base/base-item.model';
import { AuthorModelItem } from 'src/app/shared/models/author/author.model.item';
import { PrintingEditionType, PrintingEditionCurrency } from 'src/app/shared/enums';

export class PrintingEditionModelItem extends BaseItemModel {
    public title?: string;
    public description?: string;
    public authors?: Array<AuthorModelItem>;
    public price?: number;
    public type?: PrintingEditionType;
    public currency?: PrintingEditionCurrency;
    public image?: string;
}
